devlab/tests/generate_load.py

```diff
@@ -270,14 +270,14 @@ def get_vm_nics(_vm):
                     if not net['router:external'] and net['tenant_id'] == _t]
             return nics
 
-        for vm in config.vms:
+        for vm in self.config.vms:
             vm['image'] = self.get_image_id(vm['image'])
             vm['flavor'] = self.get_flavor_id(vm['flavor'])
             vm['nics'] = get_vm_nics(vm)
             self.check_vm_state(self.novaclient.servers.create(**vm))
-        for tenant in config.tenants:
-            if 'vms' in tenant:
-                for user in config.users:
+        for tenant in self.config.tenants:
+            if tenant.get('vms'):
```

Comment:
Minor: this modification is not really needed, you could leave the old code here as it's actually more readable that this.